.travis.yml

```diff
@@ -18,4 +18,4 @@ install: true
 script:
   - make .gitvalidation
   - make lint
-  - make binaries
+  - make ocid ocic
```

Comment:
you fixed conmon build issues in subsequent commits so should this be reenabled or commit take out of this series?